### PR TITLE
Support :uuid column type for validate_absence_of matcher

### PR DESCRIPTION
**Pull request by dlupu**
Hello, 

I run into a case where I needed to make sure that an `uuid` column was empty but could not use `validate_absence_of` matcher. 

It tried to set the column with "an arbitrary value" which is not a valid uuid value.

This Pull Requests attempts to fix the situation.

### CODE REVIEW

**Review by VSPPedro — DISMISSED** [review]

**Comment by VSPPedro** [comment]
Hi, @dlupu, thank you very much for your contribution!

There is only one detail: rubocop detected trailing whitespace. Would be possible to fix that, please?

About the error with 3.1.2, rails_6_0 and PostgreSQL I'll take a closer look as soon as I have some time.

**Comment by dlupu** [comment]
@VSPPedro sorry for the late reply.

I removed the rubocop issue. 

However, I'm surprised about the error that appears only on `Test / build (3.1.2, rails_6_0, postgresql)`. I've cloned the code locally and run the failing test on my machine using
* ruby 3.1.2
* appraisal rails_6_0
* postgresql db adapter

The tests passes ....

File: spec/unit/shoulda/matchers/active_model/validate_absence_of_matcher_spec.rb [hunk 1]
```diff
@@ -14,6 +14,7 @@ def self.available_column_types
         :time,
         :date,
         :binary,
+        :uuid
```
Comment (dlupu) [inline spec/unit/shoulda/matchers/active_model/validate_absence_of_matcher_spec.rb:17]:
```suggestion
        :uuid,
```

**Comment by dlupu** [comment]
@VSPPedro  it seems that `Test / build (3.1.2, rails_6_0, postgresql)` is green not. I also fixed the last rubocop issue, please trigger the CI again.

**Comment by VSPPedro** [comment]
Hi @dlupu, thanks for working on this. The failure we had earlier looks like an intermittent test. It was probably not caused by your changes. I'll keep an eye on that. Thank you again!

**Review by VSPPedro — APPROVED** [review]

**Pull request merged**